src/TitleForm/TitleForm.js

```diff
@@ -0,0 +1,243 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+import { get } from 'lodash';
+import { OnChange } from 'react-final-form-listeners';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+import {
+  Accordion,
+  AccordionSet,
+  Button,
+  Col,
+  ExpandAllButton,
+  Pane,
+  PaneFooter,
+  Paneset,
+  Row,
+  TextField,
+} from '@folio/stripes/components';
+import { Pluggable } from '@folio/stripes/core';
+import { ViewMetaData } from '@folio/stripes/smart-components';
+import {
+  useAccordionToggle,
+  validateRequired,
+} from '@folio/stripes-acq-components';
+
+import {
+  SECTIONS,
+} from './constants';
+import ProductIdDetailsForm from './ProductIdDetailsForm';
+
+const FormFooter = ({
```

Comment:
I remember someone did common component, but I couldn't find in acq-components

Reply:
it's in ui-invoice